src/main/java/com/afornalik/ox/Player.java

```diff
@@ -1,12 +1,10 @@
-package com.afornalik.ox.model.player;
+package com.afornalik.ox;
 
-import com.afornalik.ox.model.board.FieldStatus;
-
-public class Player {
+class Player {
 
     private final String name;
     private int score;
-    private final FieldStatus sign;
+    private final Field sign;
     private final boolean first;
 
     private Player(PlayerBuilder playerBuilder) {
```

Comment:
Czemu konstruktor przyjmuje budowniczego? Co to za dziwo?

Reply:
Takie rozwiązanie gdzieś widziałem. Zmieniłem.